meta-resin-common/recipes-support/resin-hostname/resin-hostname/resin-hostname.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Resin Hostname Configuration
+Requires=resin-device-uuid.service
+After=resin-device-uuid.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=@BINDIR@/%p
```

Comment:
I'm not sure we should be using %p here.

https://www.freedesktop.org/software/systemd/man/systemd.unit.html says:

"%p"	Prefix name	For instantiated units, this refers to the string before the "@" character of the unit name. For non-instantiated units, this refers to the name of the unit with the type suffix removed.

This is not an instantiated unit